src/feature/links/components/LinkItem.tsx

```
@@ -0,0 +1,45 @@
+import {Button} from "@/components/ui/button.tsx";
+import * as Icons from "react-icons/fa";
+import {LinkItemTypeSchema} from "@/feature/links/schema/linktreeSchema.ts";
+
+interface LinkItemProps {
+    link: LinkItemTypeSchema;
+}
+
+// @ts-ignore
```

ESLint

[error] Use "@ts-expect-error" instead of "@ts-ignore", as "@ts-ignore" will do nothing if the following line is error-free.

(@typescript-eslint/ban-ts-comment)